app/Main.hs

```diff
@@ -295,6 +295,18 @@ bindistInfos =
                   artifactName = "ghc-wasm-x86_64-darwin-bindist"
                 }
           },
+      (,)
+        "wasm32-wasi-ghc-gmp-aarch64-linux"
+        BindistInfo
+          { isGhcBindist = False,
```

Comment:
Only noticing this now: We currently can't set this to `True` for the GHA-built bindists for at least two reasons:

 - The GHA bindists use zstd instead of lzma, and the current ghc subdir logic only supports lzma (should be easy to fix).
 - GHA artifacts are wrapped in another `.zip` layer, so that would also have to be removed.

However, fixing this isn't a big priority; this mechanism was added in #6 mostly as convenient metadata for downstream consumers like ghcup.